testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveSparkCompatibility.java

```diff
@@ -213,6 +213,57 @@ public void testSparkParquetTimestampCompatibility(String sparkTimestampFormat,
         onSpark().executeQuery("DROP TABLE " + sparkTableName);
     }
 
+    @Test(groups = {HIVE_SPARK, PROFILE_SPECIFIC_TESTS})
+    public void testInsertFailsInBucketedTableWithoutBucketingVersionTableParameter()
```

Comment:
change to `testInsertFailsOnBucketedTableCreatedBySpark` and others correspondingly